server_admin/topics/authentication/kerberos.adoc

```diff
@@ -188,6 +188,48 @@ WARNING: Credential delegation has some security implications so only use it if
          It's highly recommended to use it together with HTTPS.
          See for example http://www.microhowto.info/howto/configure_firefox_to_authenticate_using_spnego_and_kerberos.html[this article] for more details.
 
+==== Cross-realm trust
+
+In the Kerberos V5 protocol, the `realm` is a set of Kerberos principals defined in the Kerberos database (typically LDAP servers).
+Kerberos protocol has a concept of cross-realm trust. For example if there are 2 kerberos realms A and B, the cross-realm trust
+will allow the users from realm A to access resources (services) of realm B. This means that realm B trusts the realm A.
```

Comment:
In the Kerberos V5 protocol, the `realm` is a set of Kerberos principals defined in the Kerberos database (typically LDAP servers).
The Kerberos protocol has a concept of cross-realm trust. For example, if there are 2 kerberos realms A and B, the cross-realm trust
will allow the users from realm A to access resources (services) of realm B. This means that realm B trusts the realm A.